src/writer.js

```diff
@@ -67,7 +67,10 @@ define([
             jrm = "http://dev.commcarehq.org/jr/xforms";
         }
 
-        uuid = form.formUuid; //gets set at parse time/by UI
```

Comment:
I wonder when that comment became invalid? I'm pretty sure it used to be correct.